ci/Dockerfile.datadog

```diff
@@ -3,11 +3,17 @@ FROM datadog/agent:latest
 RUN \
   DEBIAN_FRONTEND=noninteractive apt-get update \
   && apt-get install -y --no-install-recommends -o Dpkg::Options::="--force-confnew" \
-      ca-certificates jq \
+      ca-certificates jq wget openssl \
     && apt-get clean
 
-COPY ./certs/VA-Internal-S2-RCA1-v1.cer /usr/local/share/ca-certificates/VA-Internal-S2-RCA1-v1.cer
-RUN /usr/sbin/update-ca-certificates
+# Copy the host's build context directory (notification-api/) to the image's working directory (/app).
+COPY --chown=vanotify ./scripts/import-va-certs.sh /scripts/
+RUN chmod +x /scripts/import-va-certs.sh
```

Comment:
Would you please chmod it locally and push that change? It should not require this in the dockerfile.